clients/cobol-lsp-vscode-extension/src/__tests__/services/SettingsTest.spec.ts

```diff
@@ -14,32 +14,21 @@
 import * as fs from "fs-extra";
 import * as path from "path";
 import * as vscode from "vscode";
-import { C4Z_FOLDER, GITIGNORE_FILE } from "../../constants";
-import {
-  createFileWithGivenPath,
-  SettingsService,
-} from "../../services/Settings";
+import { SettingsService } from "../../services/Settings";
 import { SettingsUtils } from "../../services/util/SettingsUtils";
 
 const fsPath = "tmp-ws";
-let wsPath: string;
-let c4zPath: string;
-let filePath: string;
-
 beforeAll(() => {
   (vscode.workspace.workspaceFolders as any) = [
-    { uri: { fsPath, path: fsPath } } as any,
+    { uri: { fsPath: makefsPath(fsPath), path: makePath(fsPath) } } as any,
   ];
   const firstWorkspaceFolder = vscode.workspace.workspaceFolders![0];
   if (!firstWorkspaceFolder) return;
-  wsPath = path.join(firstWorkspaceFolder.uri.fsPath);
-  c4zPath = path.join(wsPath, C4Z_FOLDER);
-  filePath = path.join(c4zPath, GITIGNORE_FILE);
 });
 
 afterAll(() => {
-  if (fs.existsSync(wsPath)) {
-    fs.remove(wsPath);
+  if (fs.existsSync(makefsPath(fsPath))) {
+    fs.remove(makefsPath(fsPath));
   }
```

Comment:
Remove this as well

Reply:
removed : 42b9d17c478a1cb3f35c095a54dd6f5c60688007